pkgs/shells/fish/oh-my-fish/default.nix

```diff
@@ -25,31 +25,30 @@ stdenv.mkDerivation (finalAttrs: {
     fish
   ];
 
-  strictDeps = true;
-
   dontConfigure = true;
   dontBuild = true;
 
   installPhase = ''
     runHook preInstall
 
-    mkdir -pv $out/bin $out/share/oh-my-fish
-    cp -vr * $out/share/oh-my-fish
+    mkdir -pv $out/bin $out/share/${finalAttrs.pname}
+    cp -vr * $out/share/${finalAttrs.pname}
+
+    cat << EOF > $out/bin/omf-install
+    #!${runtimeShell}
 
-    cp -v ${substituteAll {
-      name = "omf-install";
-      src = ./omf-install;
-      OMF = placeholder "out";
-      inherit fish runtimeShell;
-    }} $out/bin/omf-install
+    ${fish}/bin/fish \\
+      $out/share/${finalAttrs.pname}/bin/install \\
+      --noninteractive \\
+      --offline=$out/share/${finalAttrs.pname}
 
+    EOF
     chmod +x $out/bin/omf-install
-    cat $out/bin/omf-install
 
     runHook postInstall
   '';
 
-  meta = {
+  meta = with lib; {
```

Comment:
```suggestion
  meta = {
```

Please don't add it

Reply:
There's multiple initiative going on to remove `with lib;`, see it by yourself: https://github.com/NixOS/nixpkgs/pull/293767

Reply:
hey this is a draft

Reply:
Oops. sorry !